utils/prepare-common.js

```diff
@@ -41,6 +43,34 @@ const prepareProjects = (hgPath) => {
     fs_utils.copyRecursiveSync(jQuery, path.join(netCoreDemos, 'wwwroot/js'));
     fs_utils.copyRecursiveSync(jszip, path.join(netCoreDemos, 'wwwroot/js'));
 
+    fs.readdirSync(dxDiagrammDist)
+        .filter(file => /.*\.css$/.test(file))
+        .map(file => { 
```

Comment:
better to use forEach if you do not need the new values of array

Reply:
Fixed